bioio/writers/ome_zarr_writer_2.py

```diff
@@ -452,7 +452,7 @@ def _downsample_and_write_batch_t(
         data_tczyx = data_tczyx.persist()
         # data_tczyx.compute()
         for k in range(start_t, end_t):
-            self.levels[0].zarray[k] = data_tczyx[k - start_t]
+            self.levels[0].zarray[k] = data_tczyx[k - start_t].compute()
```

Comment:
I was wondering if the compute call here is redundant with the persist call above, but it seemed to fix the unit test issue?